app/models/concerns/profile_tailoring.rb

```diff
@@ -23,4 +33,27 @@ def added_rules
   def removed_rules
     parent_profile.rules - rules
   end
+
+  def update_os_minor_version(version)
+    return unless version && os_minor_version.empty?
+
+    update!(os_minor_version: version)
+  end
+
+  private
+
+  def in_account(account, policy)
+    Profile.find_by(account: account, ref_id: ref_id,
+                    policy: policy,
+                    benchmark_id: benchmark_id)
+  end
+
+  def create_child_profile(account, policy)
+    new_profile = dup
+    new_profile.update!(account: account, parent_profile: self,
+                        external: true, policy: policy)
+    new_profile.update_rules(ref_ids: rules.pluck(:ref_id))
+
+    new_profile
+  end
```

Comment:
Similar comment: this isn't specific to tailoring, so it shouldn't be here in profile_tailoring.